frontend/src/pages/ProjectDetails.tsx

```diff
@@ -41,36 +61,241 @@ const ProjectDetailsPage = () => {
         message="Sorry, the project you're looking for doesn't exist"
       />
     )
-  const params = ['idx_updated_at', 'idx_forks_count', 'idx_stars_count', 'idx_contributors_count']
-  const filteredIcons = getFilteredIcons(project, params)
-
-  const handleButtonClick = () => {
-    window.open(`/projects/contribute?q=${project.idx_name}`)
-  }
 
-  const SubmitButton = {
-    label: 'Contribute',
-    icon: <FontAwesomeIconWrapper icon="fa-solid fa-code-fork" />,
-    onclick: handleButtonClick,
-  }
+  const toggleContributors = () => setShowAllContributors(!showAllContributors)
+  const toggleLanguages = () => setShowAllLanguages(!showAllLanguages)
+  const toggleTopics = () => setShowAllTopics(!showAllTopics)
 
   return (
-    <div className="container mx-auto pb-16 pt-24 xl:max-w-full">
-      <div className="flex justify-center">
-        <Card
-          key={project.objectID}
-          title={project.idx_name}
-          url={project.idx_url}
-          summary={project.idx_summary}
-          level={level[`${project.idx_level as keyof typeof level}`]}
-          icons={filteredIcons}
-          leaders={project.idx_leaders}
-          topContributors={project.idx_top_contributors}
-          topics={project.idx_topics}
-          button={SubmitButton}
-        />
+    <div className="mt-16 min-h-screen bg-white p-8 text-gray-600 dark:bg-[#212529] dark:text-gray-300">
+      <div className="mx-auto max-w-6xl">
+        <h1 className="mb-6 mt-4 text-4xl font-bold">{project.idx_name}</h1>
+        <p className="mb-6 text-xl">{project.idx_description}</p>
+
+        <div className="mb-8 grid grid-cols-1 gap-6 md:grid-cols-2">
```

Comment:
Can we have these cards in a different layout? I was thinking maybe we could make `Statistics` narrower, since there's not that much data in that one. And make the `Project details` block wider. Some projects have a lot of leaders for example.  So maybe 2/3 - 1/3? Or something like that..